pkg/referrerstore/oras/oras.go

```diff
@@ -40,7 +40,7 @@ import (
 
 const (
 	storeName             = "oras"
-	defaultLocalCachePath = "~/.ratify/local_oras_cache"
+	defaultLocalCachePath = ".ratify/local_oras_cache"
```

Comment:
Is there a benefit to fetch the home directory for ratify from a constant so that the same is used https://github.com/deislabs/ratify/blob/f40c0d9daa3900d90ce647bb9592454a507a93b1/config/config.go#L36. And also you wouldn't need to initialize the location with path separator which could be different for different OS?